### ISSUE TITLE
Instructions updates

### ISSUE THREAD

**Issue by brightbold** [North101/mpc_projects#69]
@Vitus Prem on BGG gave the following documentation feedback:

> 3. the section regarding „Downloading a project“ left me confused.
> I had just installed the add-on and opened the extra window on the mpc website, so I kind of figured I should be seeing the projects and download-buttons right there. Took me embarrassingly long to figure out they were on another part of  Inexorable Fate.
> A link to the projects page at the beginning of the section on downloading projects would have helped me greatly.
> 
> 4. Again, once I found the projects page, I was re-excited. So beautiful, so well made! Picking projects and combining them was well explained and easy to handle.
> 
> 5. Next, I was a little sad when I noticed how many projects came with broken images … Arkham Incidents, Nameless City, Alt-Art permanents, Onigawa, to name a few. Maybe lucky too, as mpc drained my bank account even with the „reduced“ order.
> I only read about the issue with broken images in the aftermath; maybe a hint about this would be great somewhere in the beginners instructions, too?

**Comment by dancuadra** [comment]
I added a Website Help tab in the [Inexorable Fate Project Prep](https://docs.google.com/document/d/1gFG7qrNIintDnv-QEHWcgLUz_3-GCO-Q5rwZNm8pWys/edit?tab=t.6os4jla0thx4#heading=h.s20ic3mrb4ol) google doc with a number of redlines I recommend for the help page.

To address the confusion about the **Downloading a Project** section, I removed the instructions about navigating to MPC and opening the extension for the first time from the **Installing the Extension** section to the beginning of the **Load a Project File** section. The instructions about navigating InexF and finding a .json file should come before the user ever opens MPC.

Re: the broken links issue, the existing page refers to contacting the project creator to initiate refreshing the cache if there is an issue. I think we shouldn't encourage people to directly contact content creators, so I suggest we change that to contact InexF support and/or @coldtoes, and our team then facilitate updates. @brightbold What do you think? Is there a better workflow for handling this?

Also, I recommend we take out the section about methods for adding bleed. The instructions for Strange Eons are confusing and I think we'd need to provide a lot more direction if we want to cover using SE at all (like having a dedicated page about how to export from SE with Zoop etc.) We could never get the bleedy.py script to work when I was writing our internal instructions so I think we should take that out of the public facing page unless we can provide better troubleshooting.

Lastly, I removed the "content creator" sections for **Combining and Editing** and **Viewing Projects**. These are redundant and should just link back to the same section earlier in the page.

**Comment by brightbold** [comment]
> Re: the broken links issue, the existing page refers to contacting the project creator to initiate refreshing the cache if there is an issue. I think we shouldn't encourage people to directly contact content creators, so I suggest we change that to contact InexF support and/or @coldtoes, and our team then facilitate updates. @brightbold What do you think? Is there a better workflow for handling this?

This is old language from when we didn't realize all the issues with refreshing the cache and thought people might be making projects and sharing them with others. So that wasn't supposed to mean the homebrewer but instead the person who sent you the json file. Now I think it should specify coldtoes for English content and founchopf (Founi) for German.

**Comment by brightbold** [comment]
> Also, I recommend we take out the section about methods for adding bleed. The instructions for Strange Eons are confusing and I think we'd need to provide a lot more direction if we want to cover using SE at all (like having a dedicated page about how to export from SE with Zoop etc.) We could never get the bleedy.py script to work when I was writing our internal instructions so I think we should take that out of the public facing page unless we can provide better troubleshooting.

This SE stuff is the stuff I want in the content creator section. I'm thinking this type of content falls into two sections (the second of which already exists): 

1. You're a content creator, you have a project in SE:

  - you need to know how to install Arno and Teutonic/Arkhamic 
  - you need to know how to use Zoop for numbering, bleed, export/naming (most of this can point to Zoop documentation)

2. You are a person who has pngs of cards someone else created and you want to print them:

  - you need to use Bleedy UI to add bleed
  - you have to manually rename the files so they work with the extension.

Thoughts?

I just poked Dangeroo (nee Buteremelse) about Bleedy.UI, which is here: https://thankful-mushroom-08ecc5d1e.4.azurestaticapps.net/ and if he can clean up some of the issues this will be much easier for people — no installing python or having to use the terminal or whatnot. So I will keep irritating him until he gets it in usable condition, but let's refer to that instead of the local script.

**Comment by brightbold** [comment]
> Lastly, I removed the "content creator" sections for Combining and Editing and Viewing Projects. These are redundant and should just link back to the same section earlier in the page.

The reason I duplicated this was because I think people feel this page is overwhelming, so I was trying to put everything in discrete sections so people could focus on a smaller part. Do you have suggestions as to how we could address this concern? One of my thoughts was to accordion it like the FAQ page. But I'm not sure what the best course of action is here and would love your thoughts.

**Comment by dancuadra** [comment]
>This is old language from when we didn't realize all the issues with refreshing the cache and thought people might be making projects and sharing them with others. So that wasn't supposed to mean the homebrewer but instead the person who sent you the json file. Now I think it should specify coldtoes for English content and founchopf (Founi) for German.

Makes sense. So we need to include contact info for the relevant people.

I like the outline of SE content to include. If this is going to be a new page, then maybe the best strategy is to break up the Help page into multiple smaller pages. That would also address the issue of the page feeling overwhelming. For example:

- Print an Existing Project
- Print a New Project, divided into "I have a card images I want to print" and "I have a project created in Strange Eons"

>Do you have suggestions as to how we could address this concern? One of my thoughts was to accordion it like the FAQ page.

In general the accordion effect is helpful for all these pages that are text heavy.

>I just poked Dangeroo (nee Buteremelse) about Bleedy.UI, which is here: https://thankful-mushroom-08ecc5d1e.4.azurestaticapps.net/ and if he can clean up some of the issues this will be much easier for people — no installing python or having to use the terminal or whatnot. So I will keep irritating him until he gets it in usable condition, but let's refer to that instead of the local script.

That's awesome, I didn't know there was a web based version of this. That's way more accessible than using python. I do wish it had a more professional looking appearance though.

**Comment by brightbold** [comment]
> That's awesome, I didn't know there was a web based version of this. That's way more accessible than using python. I do wish it had a more professional looking appearance though.

Yeah it's pretty new. Do you mean more professional in terms of the font choice and casual sketch aesthetic, or in terms of things like the words "Settings" and "Select" breaking randomly and the buttons at the bottom being off-center. Because things in that latter category will get fixed.

**Comment by dancuadra** [comment]
Yeah, I was referring to the font and aesthetic. I assume the functionality is a work in progress.

**Comment by dancuadra** [comment]
@brightbold What's the next step for this? My document is complete except for the Adding Bleed in SE section. Is there anything else I should do or is it in your hands to update the website end?

**Comment by brightbold** [comment]
@dancuadra Aargh. Sorry. I missed the comment above and than have just let this languish. I will get your updates in once I'm able to get stuff to deploy again.
